GameData/B9_Aerospace_HX/Parts/Structure_HX/model_hpd1.cfg

```diff
@@ -291,51 +217,52 @@ PART
         {
             name = Oxidizer
             ratio = 1.1
+            DrawGauge = True
         }
         atmosphereCurve
         {
-            key = 0 323 // 308 * 1.05
-            key = 1 281 // 268 * 1.05
+            key = 0 900
+            key = 1 600
+			key = 9 0.01
         }
     }
-
     MODULE
     {
         name = ModuleEnginesFX
-        engineID = HybridPlasma
+        engineID = SupraNuclear
+        runningEffectName = running_lava
         powerEffectName = power_plasma
         thrustVectorTransformName = nozzle_transform
         exhaustDamage = true
         ignitionThreshold = 0.1
         minThrust = 0
-        maxThrust = 4280 // 4761 * 0.9
-        heatProduction = 250
+        maxThrust = 9000
+        heatProduction = 300
         useEngineResponseTime = True
         engineAccelerationSpeed = 1.60
         engineDecelerationSpeed = 2
+		exhaustDamageMultiplier = 2400
+		exhaustDamageMaxRange = 200
         PROPELLANT
-        {
-            name = ElectricCharge
-            ratio = 16.5
-        }
-        PROPELLANT
+		{
+			name = ElectricCharge
+            ratio = 45
+            DrawGauge = True
+		}
+		PROPELLANT
         {
             name = LiquidFuel
-            ratio = 0.9
+            ratio = 1
             DrawGauge = True
         }
-        PROPELLANT
-        {
-            name = Oxidizer
-            ratio = 1.1
-        }
         atmosphereCurve
         {
-            key = 0 840 // 800 * 1.05
-            key = 1 230 // 220 * 1.05
+            key = 0 2000
+            key = 1 900
+			key = 4 0.01
         }
     }
-
+	
```

Comment:
🤔

Reply:
SupraNuclear is an NTR/plasma/hybrid engine mode. afaik it tends to easily flameout in high atmosphere pressure. I didn't think to buff it with respect to this engine's size. Also, I wasn't aware the indentation here is a load of spaces.

Reply:
Tab added to empty line